sklearn/linear_model/setup.py

```diff
@@ -1,7 +1,9 @@
 import os
-
 import numpy
 
+from sklearn._build_utils import gen_from_templates
```

Comment:
out of curiosity, can you explain me why an absolute import is required here ?

Reply:
It doesn't work if we use a relative import?

Reply:
No, they also did it for the deprecated modules in sklearn/setup.py
Maybe @thomasjpfan knows better about that ?